plugins/org/src/components/Cards/OwnershipCard/ComponentsGrid.tsx

```diff
@@ -75,6 +75,7 @@ const EntityCountTile = ({
         display="flex"
         flexDirection="column"
         alignItems="center"
+        style={{ height: '100%' }}
```

Comment:
Can we add this not to one of the classes? 🤔

Reply:
Agree, pure laziness on my part should have refactored once I found a solution. I've moved this to one of the classes now.